### PR TITLE
cozy: clean up and misc fixes

### PR DESCRIPTION
**Pull request by Aleksanaa**
## Description of changes

Discovered this package when digging into https://github.com/NixOS/nixpkgs/pull/301865, but this one is just too... <I don't know how to comment>

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by Cryolitia — APPROVED** [review]
LGTM

小帕鲁辛苦了

File: pkgs/applications/audio/cozy/default.nix [hunk 1]
```diff
@@ -1,52 +1,43 @@
...
     gobject-introspection
   ];
 
   buildInputs = [
-    gtk3
-    cairo
-    gettext
-    gnome.adwaita-icon-theme
     libadwaita
     libdazzle
```
Comment (jtojnar) [inline pkgs/applications/audio/cozy/default.nix:40]: Libdazzle is GTK 3 library too.

File: pkgs/applications/audio/cozy/default.nix
Comment (jtojnar) [inline pkgs/applications/audio/cozy/default.nix:1]: Might be a good idea to move it to by-name while at it.

File: pkgs/applications/audio/cozy/default.nix [hunk 2]
```diff
@@ -1,52 +1,43 @@
...
+, gst_all_1
 }:
 
 python3Packages.buildPythonApplication rec {
-
-  format = "other"; # no setup.py
-
   pname = "cozy";
   version = "1.3.0";
+  pyproject = false; # no setup.py
```
Comment (jtojnar) [inline pkgs/applications/audio/cozy/default.nix:19]: This comment looks outdated.

File: pkgs/applications/audio/cozy/default.nix [hunk 3]
```diff
@@ -57,7 +48,6 @@ python3Packages.buildPythonApplication rec {
 
   propagatedBuildInputs = with python3Packages; [
     apsw
-    cairo
     dbus-python
```
Comment (jtojnar) [inline pkgs/applications/audio/cozy/default.nix:51]: All are of these still necessary? Either the [upstream list](https://github.com/geigi/cozy/blob/027efb7622ef38795d3cc006fb0704e3a8f86ba5/meson.build#L41) is outdated or this one is.

File: pkgs/applications/audio/cozy/default.nix
Comment (Aleksanaa) [reply pkgs/applications/audio/cozy/default.nix:1]: I don't want to move these files manually just yet. Give infinisil a little more time

**Review by makefu — APPROVED** [review]
Changes Look good to me and software starts up with the changes.

**Pull request merged**